ui/components/src/model/units/bytes.ts

```diff
@@ -31,16 +34,37 @@ export const BYTES_UNIT_CONFIG: Readonly<Record<BytesUnitKind, UnitConfig>> = {
 };
 
 // https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript/18650828#18650828
-export function formatBytes(bytes: number, decimals = 2) {
+export function formatBytes(bytes: number, unitOptions: BytesUnitOptions) {
+  // default to full 'Bytes' formatting
+  const options = unitOptions.abbreviate === undefined ? { ...unitOptions, abbreviate: false } : unitOptions;
```

Comment:
I wonder if it would be nice if this default was built into the form itself? 

So like, I imagining a radio button with only two choies, and it defaults to false. Then, the value is always either true or false, so we don't need to do this.

But I dunno, kinda gets into "what's best way to handle defaults", and I think it's nice that in this code both defaults are being handled together in this same function

Reply:
I might be misunderstanding, but I think you have to handle it in the code since users could just edit the JSON and aren't necessarily using form controls in panel edit drawer? There might be a better way to do this though so talking through the best way to handle defaults sounds great since it isn't consistent currently.

Reply:
Ahhhhh very good point, I hadn't thought of that